docs/read-more/how-to-write-a-sequence.md

```diff
@@ -0,0 +1,221 @@
+
+Sequence is a program that produces, consumes or transforms data. It’s a function or an array of functions. They typically look somewhat like this:
+
+```ts
+/** The function parameters are
+* input stream
+* ...params passed to instance on start
+*/
+export default function(input, param1, param2) {
+    const out = new PassThrough()
+
+    input.on('data', (data) => {
```

Comment:
Let's put asnyc iteration version higher. Also we need flow control in this sample

Reply:
fixed